### ISSUE TITLE
v0.5.8 don't show loadingmore view at the first time

### ISSUE THREAD

**Issue by julist213** [cymcsg/UltimateRecyclerView#387]
first load -> var  enabled_custom_load_more_view =false  in UltimateViewAdapter and method internalExecuteLoadingView() exec at the time pull to bottom ,so adapter could't getFooterView at the first time ?what's wrong.

**Comment by jjhesk** [comment]
it was set to be this way. if you like to open it, you could try to override in at the oncreate part in the adapter.

**Comment by julist213** [comment]
@jjhesk i solved problem  by dynamic changing  the boolean value of var enabled_custom_load_more_view .by the way why ,in DataObserver method  updateHelperDisplays() do mEmpty.setVisibility(mEmptyView == null ? View.VISIBLE : View.GONE) , rather than mEmpty.setVisibility(mEmptyView == null ? View.GONE: View.VISIBLE) ?thx

**Comment by jjhesk** [comment]
yes you can certainly do so. there is a better way if you would looking at the sample app. try to learn from "debug first page cancel load more". You may download from [here](https://github.com/cymcsg/UltimateRecyclerView/releases)

**Comment by julist213** [comment]
@jjhesk that demo can handle disable and reenable loadmore but it's no use to show loadmore view at first time ,even every loadmore action loadmore view can't display.

**Comment by passerbyABC123** [comment]
@julist213 Facing the same problem here. Might you elaborate more on how you solved that?

**Comment by julist213** [comment]
@passerbyABC123 i do in my adapter like:
public ...Adapter(){
        enabled_custom_load_more_view=true;//for first time showing?
    }
//open this method to  dynamic change the boolean value ?
    public void setLoadMoreEnable(boolean enable){
        enabled_custom_load_more_view=enable;
    }
it's work for me ,can help you ?

**Comment by jjhesk** [comment]
enabled_custom_load_more_view is not for the first time showing and setLoadMoreEnable is an internal control method.
